lib/ex_machina/ecto.ex

```diff
@@ -66,6 +73,50 @@ defmodule ExMachina.Ecto do
     |> drop_ecto_fields
   end
 
+  @doc """
+  Same as `params_for/2`, but it inserts all belongs_to associations and sets the
```

Comment:
Maybe this could be shortened to `Same as `params_for/2`, but inserts all belongs_to associations and sets the foreign keys`? If you don't like that I'm fine with it as-is as well